auth/realm/base/src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -640,7 +696,7 @@ private Void createPrivileged() throws RealmUnavailableException {
                         streamWriter.writeCharacters("\n");
                         streamWriter.writeStartElement("identity");
                         // Continue to write using 1.0 as not using any features added in 1.0.1
-                        streamWriter.writeDefaultNamespace(ELYTRON_1_0);
+                        streamWriter.writeDefaultNamespace(Version.VERSION_1_0.getNamespace());
```

Comment:
When `secretKey` is not null, we should be writing version 1.1.

Reply:
The schema doesn't change when using an encrypted realm. Would this still require a Namespace bump?

Reply:
Ah, are the schema changes only for the integrity portion of the changes? If so, ignore this.

Reply:
Actually, was just thinking some more, the format value "enc_base64" is only allowed in the new 1.1 schema. This isn't a valid attribute for 1.0. So I think we should make the suggested changes.

Reply:
Changed